docs/guides/gunw_product_guide.md

```diff
@@ -146,6 +146,7 @@ will help ensure that you find results that match the desired [ARIA Frame ID](#a
       dual-pol and single-pol modes in the search results
 - **Direction**: match the orbit direction of the [ARIA Frame ID reference geojson](#aria-frame-id-maps) 
   used to select the desired ARIA Frame (or reference the `dir` attribute from the ARIA Frame ID geojson file)
+- **Subtype**: Select both `SA` and `SB` (ARIA S1 GUNW products from Sentinel-1C acquisitions are not supported)
```

Comment:
```suggestion
- **Subtype**: Select both `SA` and `SB` (ARIA S1 GUNW products from Sentinel-1C acquisitions are not currently supported)
```

Reply:
Also, both this one and the polarization note have a capital after the colon. Would be better to keep it consistent and edit the polarization entry as well.